### PR TITLE
Removed dprintf from perf library

### PR DESCRIPTION
**Pull request by katzfey**
## Describe problem solved by this pull request
Removed dprintf from perf library since it is only ever used with fd=1 (STDOUT). This helps with some platforms (e.g. Qurt) which have some Posix support but not full Posix support.

## Describe your solution
Changed from dprintf to PX4_INFO

## Describe possible alternatives
Could have made some platform specific changes to allow dprintf but that will just add more special pound defines.

## Test data / coverage
Tested on VOXL 2

### CODE REVIEW

**Comment by dagar** [comment]
Can you use PX4_INFO_RAW? Then it skips logging and the module prefix isn't appended.

Screenshot from 2022-10-19 20-34-46

**Comment by katzfey** [comment]
Changed PX4_INFO to PX4_INFO_RAW

**Comment by dagar** [comment]
Thanks, output looks the same now.

**Review by dagar — DISMISSED** [review]

File: src/lib/perf/perf_counter.cpp [hunk 1]
```diff
@@ -37,6 +37,10 @@
  * @brief Performance measuring tools.
  */
 
+#ifndef MODULE_NAME
+#define MODULE_NAME "perf_counter"
```
Comment (dagar) [inline src/lib/perf/perf_counter.cpp:41]: No longer needed?

Comment (katzfey) [reply src/lib/perf/perf_counter.cpp]: Nope, not needed. Removed.

**Review by dagar — APPROVED** [review]

**Pull request merged**